### ISSUE TITLE
logging: Missing verb.

### ISSUE THREAD

**Issue by JoeGruffins** [decred/dcrdex#1770]
Extremely minor but this needs a %v:
```
trade.go:1273:          t.dc.log.Tracef("Match %s not redeemable: ticks metered", match, match.swapErr)
```

**Comment by JoeGruffins** [comment]
Tagging another logging thing here.

These back slashes around the address look unintended.
```
2022-08-15 04:38:24.123 [WRN] CORE: ltc wallet communication issue: "Post \"http://127.0.0.1:6789\": context deadline exceeded"
```
Can also see in the UI:
![image](https://user-images.githubusercontent.com/34998433/184580732-df22de14-168d-4f72-90ed-915b39265e0f.png)

**Comment by ukane-philemon** [comment]
> These back slashes around the address look unintended.

I think we can leave this one as it is? `err` could be anything. `err` in this case contains a quote hence the escape with back slashes.
```
func (c *Core) peerChange(w *xcWallet, numPeers uint32, err error) {
	if err != nil {
		c.log.Warnf("%s wallet communication issue: %q", unbip(w.AssetID), err.Error())
```

**Comment by JoeGruffins** [comment]
I think you are right. After following the error back, I think it is just how the error is encoded for the litecoin electrum client and not something we should try to fix.

https://github.com/decred/dcrdex/blob/d85cdad350b1699556341212bec228ff9d2a5198/client/asset/btc/electrum/wallet.go#L94-L96

So, please ignore.

**Comment by chappjc** [comment]
I think the `%q` is escaping the quotes unnecessarily.

```diff
diff --git a/client/core/locale_ntfn.go b/client/core/locale_ntfn.go
index b917d9dc..0edfccdd 100644
--- a/client/core/locale_ntfn.go
+++ b/client/core/locale_ntfn.go
@@ -57,7 +57,7 @@ var enUS = map[Topic]*translation{
        // [asset name, error message]
        TopicWalletCommsWarning: {
                subject:  "Wallet connection issue",
-               template: "Unable to communicate with %v wallet! Reason: %q",
+               template: `Unable to communicate with %v wallet! Reason: "%v"`,
        },
        // [asset name]
        TopicWalletPeersWarning: {
```

**Comment by chappjc** [comment]
https://go.dev/play/p/KNG7pAY5W2L

**Comment by JoeGruffins** [comment]
Oh, did chap fix it?

**Comment by ukane-philemon** [comment]
@chappjc, this was fixed by #1803.
